README-new.rst

```diff
@@ -0,0 +1,30 @@
+What is MultiGeiger?
+--------------------
+
+The MultiGeiger is a radioactivity measurement device.
+
+It has a lot of interesting properties, which you can find on the first page of the docs.
+
+Documentation
+~~~~~~~~~~~~~
+
+- https://multigeiger.readthedocs.org/  (english + deutsch)
+
+  This is our new, versioned and translated documentation (see the box at the
+  lower right there to switch languages and documentation versions)!
+
+  Das sind unsere neuen, versionierten und übersetzten Dokus (siehe die Box
+  rechts unten, mit der man Sprache und Version umschalten kann)!
+
```

Comment:
...was ich immer sehr hilfreich finde, sind auch direkte Links zu den Kapiteln zur Installation / Inbetriebnahme.
Können wir aber auch später noch ergänzen.
Uppsi! I think direct links to setup / installation are always helpful. They can be added at a larger stage, though.

Reply:
es gibt auf rtd links ne navigation zu den div. kapiteln der doku.

ist noch sehr ausbaubar, das hier ist nur mal ein anfang.